### PR TITLE
Adding ability to specify ulimits nofile to conatiner and also adding sensible default

### PR DESCRIPTION
**Pull request by jameshowardwang**

### CODE REVIEW

File: aegea/batch.py [hunk 1]
```diff
@@ -237,6 +237,9 @@ def ensure_job_definition(args):
             container_props["mountPoints"].append({"sourceVolume": "vol%d" % i, "containerPath": guest_path})
     iam_role = ensure_iam_role(args.job_role, trust=["ecs-tasks"],
                                policies=["AmazonEC2FullAccess", "AmazonDynamoDBFullAccess", "AmazonS3FullAccess"])
+    if args.ulimits_nofile:
+        container_props.update(ulimits=[])
```
Comment (kislyuk) [inline aegea/batch.py]: Consider changing this to `container_props.setdefault("ulimits", [])`

**Comment by kislyuk** [comment]
By the way, there are other settable ulimits (`core | cpu | data | fsize | locks | memlock | msgqueue | nice | nofile | nproc | rss | rtprio | rttime | sigpending | stack`). Perhaps a `--ulimits` arg with `nargs="*"` can be used like this:
```
parser.add_argument("ulimits", nargs="*", help="Separate ulimit name and value with colon, for example: --ulimits nofile:20000")
...
container_props.setdefault("ulimits", [])
for ulimit in args.ulimits:
    name, value = ulimit.split(":", 1)
    container_props["ulimits"].append(dict(name=name, hardLimit=value, softLimit=value))
```
`aegea batch submit --ulimits nofile:20000 nproc:9000`

**Comment by jameshowardwang** [comment]
Updated with requested changes, thanks @kislyuk

**Comment by kislyuk** [comment]
@jameshowardwang, I merged and released this as v1.10.0. I also added back in a default value for the option (`default=["nofile:100000"]`).

**Pull request merged**